src/Artsy/Relay/createEnvironment.ts

```diff
@@ -82,12 +83,15 @@ export function createEnvironment(config: Config = {}) {
             }
           : headers,
       }),
-      relaySSRMiddleware.getMiddleware({
-        lookup: true,
```

Comment:
This was added a while back to get around an issue we were facing with the router - namely, a user lands on a page that is ssr rendered and rehydrated on the client, and then navigates to another route in the app and then navigates back. On navigating back, even though the page has already been visited (first place the user lands) a new request is made. This fixes that by looking up the request. When combined with `cacheMiddleware` though the better course of action is to populate cacheMiddleware on mount. 

In terms of found-relay, I believe this is now irrelevant as a new [`dataFrom`](https://github.com/relay-tools/found-relay#datafrom) feature was added recently.

Reply:
I’m not entirely sure I understand the cause of the previous issue, but `dataFrom` is a prop that control usage from data available in the store, not eg if network cache is available (because that’s external to Relay).